### PR TITLE
Mention required nuget package in remote app doc

### LINKED ISSUE #28976: Add missing NuGet package
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 89915845-b905-fe63-0885-acc5a9392679
* Version Independent ID: 89915845-b905-fe63-0885-acc5a9392679
* Content: [Remote app setup](https://learn.microsoft.com/en-us/aspnet/core/migration/inc/remote-app-setup?branch=pr-en-us-28966&view=aspnetcore-7.0)
* Content Source: [aspnetcore/migration/inc/remote-app-setup.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/migration/inc/remote-app-setup.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-migration**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---
[Associated WorkItem - 85492](https://dev.azure.com/msft-skilling/Content/_workitems/edit/85492)

### PR DESCRIPTION
**Pull request by explunit**
I can't find anywhere in the current docs where it says to install this package. Just using the sample code will yield `The name 'SystemWebAdapterConfiguration' does not exist in the current context`

***EDIT by @Rick-Anderson***
Fixes #28976

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/migration/inc/remote-app-setup.md](https://github.com/dotnet/AspNetCore.Docs/blob/33f0ba20b98e21e56508f7e606b673e22a103f31/aspnetcore/migration/inc/remote-app-setup.md) | [Remote app setup](https://review.learn.microsoft.com/en-us/aspnet/core/migration/inc/remote-app-setup?branch=pr-en-us-28966) |

### CODE REVIEW

**Comment by explunit** [comment]
@dotnet-policy-service agree

File: aspnetcore/migration/inc/remote-app-setup.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ To enable the ASP.NET Core app to communicate with the ASP.NET app, it's necessa
 
 ### ASP.NET app configuration
 
-To set up the ASP.NET app to be able to receive requests from the ASP.NET Core app, call the `AddRemoteApp` extension method on the `ISystemWebAdapterBuilder` as shown here.
+To set up the ASP.NET app to be able to receive requests from the ASP.NET Core app:
+1. Install the nuget package `Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices`
+2. Call the `AddRemoteApp` extension method on the `ISystemWebAdapterBuilder` as shown here.
```
Comment (Rick-Anderson) [inline aspnetcore/migration/inc/remote-app-setup.md:27]:
```suggestion
2. Call the `AddRemoteApp` extension method on the `ISystemWebAdapterBuilder`:
```

File: aspnetcore/migration/inc/remote-app-setup.md [hunk 2]
```diff
@@ -22,7 +22,9 @@ To enable the ASP.NET Core app to communicate with the ASP.NET app, it's necessa
 
 ### ASP.NET app configuration
 
-To set up the ASP.NET app to be able to receive requests from the ASP.NET Core app, call the `AddRemoteApp` extension method on the `ISystemWebAdapterBuilder` as shown here.
+To set up the ASP.NET app to be able to receive requests from the ASP.NET Core app:
+1. Install the nuget package `Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices`
```
Comment (Rick-Anderson) [inline aspnetcore/migration/inc/remote-app-setup.md:26]:
```suggestion
1. Install the nuget package [`Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices`](https://www.nuget.org/packages/Microsoft.AspNetCore.SystemWebAdapters)
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Thanks for the contribution. Be sure to check out your name as a contributor about the 3rd of next month on the [What's new page](https://docs.microsoft.com/en-us/aspnet/core/whats-new/) in the **Community contributors** section at the bottom.

**Pull request merged**